test/api/templates/api_test.go

```diff
@@ -119,11 +119,11 @@ func RunE2ETest(t *testing.T, testDir, workingDir string, binary string) {
 	apiUrl, stdout, stderr := RunApiServer(t, binary, project.StorageAPIHost(), repositories)
 
 	// Assert
-	RunRequests(t, envProvider, testDirFs, workingDirFs, apiUrl)
+	requestsOk := RunRequests(t, envProvider, testDirFs, workingDirFs, apiUrl)
```

Comment:
To som pridal do `Buffer API`, tak som to dal rovno aj do `Templates API` testov.

Ked spadne request, tak dalsie kontroly sa nerobia, aby bolo viac zrejme, kde je chyba.